tor_ocr/main.py

```diff
@@ -76,8 +67,33 @@ def chunks(s, n):
         yield s[start:(start + n)]
 
 
-def run(config):
+# stolen from https://github.com/Zaargh/ocr.space_code_example/blob/master/ocrspace_example.py
+def ocr_space_url(url, overlay=False, api_key=os.getenv("OCR_API_KEY", "helloworld"), language='eng'):
+    """ OCR.space API request with remote file.
```

Comment:
Please match the docstring formatting to this: https://github.com/TranscribersOfReddit/TranscribersOfReddit/blob/master/tor/core/user_interaction.py#L118-L129